testing/dbtest/template.go

```diff
@@ -0,0 +1,111 @@
+package dbtest
+
```

Comment:
Any thoughts on writing unit tests for these funcs?

Reply:
Added some unit tests for this package.